src/askui_runner/project_template/helper/jest.setup.ts.jinja

```diff
@@ -27,7 +33,10 @@ beforeEach(async () => {
 afterEach(async () => {
   await aui.stopVideoRecording();
   const video = await aui.readVideoRecording();
-  AskUIAllureStepReporter.attachVideo(video);
+  const size = getByteSizeInMB(video);
+  if (size < 9.9) {
```

Comment:
Why is it `< 9.9`? I would have expected `<= 10`.

Reply:
adjusted, to <10 to be sure that this works.